content/v2.0/reference/cli/influx/telegrafs/update.md

```diff
@@ -0,0 +1,47 @@
+---
+title: influx telegrafs update
+description: >
+ placeholder
+menu:
+  v2_0_ref:
+    name: influx telegrafs update
+    parent: influx telegrafs
+weight: 201
+---
+
+The `influx telegrafs update` command updates a Telegraf configuration to match the specified parameters.
+If a name or description are not provided, they are set to an empty string.
```

Comment:
Interesting, so the existing name or description wouldn't be retained?

Reply:
Currently, no, they wouldn't (as far as I understand) @jsteenb2 👆 ?